### PR TITLE
ceph: try to use monitor initial members from the public network

### PR DESCRIPTION
**Pull request by mjura**
This PR is replacing https://github.com/crowbar/crowbar-ceph/pull/84

### CODE REVIEW

**Review by sidthe — APPROVED** [review]

File: chef/cookbooks/ceph/recipes/mon.rb [hunk 1]
```diff
@@ -142,13 +149,14 @@
...
 execute "Create Ceph client.admin key when ceph-mon is ready" do
-  command "ceph-create-keys -i #{node['hostname']}"
+  command "ceph-create-keys -i #{mon_name}"
   not_if { File.exists?("/etc/ceph/#{cluster}.client.admin.keyring") }
 end
 
 get_mon_addresses.each do |addr|
   execute "peer #{addr}" do
-    command "ceph --admin-daemon '/var/run/ceph/ceph-mon.#{node['hostname']}.asok' add_bootstrap_peer_hint #{addr}"
+    command "ceph --admin-daemon '/var/run/ceph/ceph-mon.#{mon_name}.asok' " \
```
Comment (dirkmueller) [inline chef/cookbooks/ceph/recipes/mon.rb:154]: you're changing the asok path here but don't adjust all other references in ceph/libraries/default.rb and other places

**Review by rsalevsky — CHANGES_REQUESTED** [review]
Can you please fix the hound issues?

File: chef/cookbooks/ceph/recipes/mon.rb (hunk 1 above)
Comment (mjura) [reply chef/cookbooks/ceph/recipes/mon.rb:154]: Done

**Comment by mjura** [comment]
@rsalevsky: I fixed Hound bitching comments :)

**Comment by mjura** [comment]
Let

**Comment by mjura** [comment]
Let's continue in https://github.com/crowbar/crowbar-ceph/pull/84

**Comment by mjura** [comment]
Let's continue in https://github.com/crowbar/crowbar-ceph/pull/84

**Pull request closed without merging**